Makefile.dep

```diff
@@ -884,6 +884,16 @@ ifneq (,$(filter riotboot_hdr, $(USEMODULE)))
   USEMODULE += riotboot
 endif
 
+ifneq (,$(filter tinydtls_sock_dtls, $(USEMODULE)))
+    USEPKG += tinydtls
+    USEMODULE += sock_dtls
+endif
+
+ifneq (,$(filter sock_dtls, $(USEMODULE)))
+    USEMODULE += credman
+    USEMODULE += sock_udp
```

Comment:
Since DTLS does not only work over UDP, I don't think this should be a dependency.

Reply:
Sorry, forgot about https://github.com/RIOT-OS/RIOT/pull/11909/files#diff-cce49629fe14fa1665fd3fdc748182a4R69 so nvm.